eightpoints/guzzle-bundle/7.0/config/packages/eight_points_guzzle.yaml

```diff
@@ -0,0 +1,2 @@
+eight_points_guzzle:
+    clients: ~
```

Comment:
Is this really needed? I just looked at the code quickly, it seams like the bundle will have the exact same behaviour without these two lines, is that correct?

Reply:
yep, behaviour the same, but user will know where to add clients...